helm/templates/tests/e2etests.yaml

```diff
@@ -62,14 +62,22 @@ spec:
               value: '{{- include "dnastack.indexingService.resourceUrl" . -}}/'
             - name: E2E_PUBLISHER_DATA_RESOURCE_URI
               value: '{{- include "dnastack.publisherData.resourceUrl" . -}}/'
-            - name: DD_CIVISIBILITY_ENABLED
-              value: "true"
-            - name: DD_ENV
-              value: "dnastack-us-central-cluster_{{ coalesce .Values.cloud.clusterNamespace ((.Values.global).cloud).clusterNamespace }}"
-            - name: DD_SERVICE
-              value: "{{ .Values.app.name }}"
-            - name: DD_TRACE_AGENT_URL
-              value: "http://10.31.241.233:8126"
```

Comment:
Should these be removed? I think @prajjwolmondal is still working on adding test monitoring. I think the better place to add it (if you remove it from here) would be in the optional test.env section in the values file but it's not there either.

Reply:
Yes Jabran and I handled that as part of this commit: https://github.com/DNAstack/cloud-deployments/commit/9fc11f894f7202be288a216b8102afd90a51cd77